.env.example

```diff
@@ -0,0 +1,5 @@
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
```

Comment:
Could you add the instruction of getting the id and secret in the README?

Reply:
updated with  [this commit](https://github.com/CSC510-GROUP-40/FitnessApp/pull/3/commits/b85108288689e41391e322ce05e76894fea3ee5a)